src/OpenApi/Serializer/OpenApiNormalizer.php

```diff
@@ -72,7 +74,7 @@ private function recursiveClean(array $data): array
      */
     public function supportsNormalization(mixed $data, string $format = null, array $context = []): bool
     {
-        return self::FORMAT === $format && $data instanceof OpenApi;
+        return \in_array($format, [self::FORMAT, self::JSONOPENAPI_FORMAT, self::YAMLOPENAPI], true) && $data instanceof OpenApi;
```

Comment:
As this is on the hot path, I would use the `||` operator instead of `in_array()` (micro-optimization).